NMS-16361: Update Kafka Producer docs (#7124)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/kafka-producer/configure-kafka.adoc

To enable metric forwarding, set the value of the `forward.metrics` property to `true`.

> The producer's code sends a full CollectionSet per message to Kafka.

Reviewer: This may be updated to `Kafka producer tries to send full CollectionSet response as one message in Kafka however If the message size is over 1MB , they will broken up into smaller messages that are less than 1MB.  When sending one CollectionSet over multiple messages, their timestamp will remain same.

Reviewer 2: ```suggestion
Kafka Producer sends a full CollectionSet as one message to Kafka.
```

Large network equipment might have hundreds of thousands of metrics that could generate messages over the Kafka limit (1MB in size by default).

> If the messages are over 1MB in size, they will be broken up into smaller messages that are identified by the original timestamp.

Reviewer 2: ```suggestion
If the message size is over 1MB , they will broken up into smaller messages that are less than 1MB.
When sending one CollectionSet over multiple messages, their timestamp will remain same.
```
